pom.xml

```diff
@@ -254,19 +255,32 @@
               java.net.http.*
             </ignores>
           </configuration>
-          <executions>
+          <!--<executions>
```

Comment:
Until we officially drop Java 8 support, I think this is required, but it might not be since we have a Java 8 test run. 
But if you're going to remove the only execution, remove the whole `<plugin>` rather than commenting out.

Reply:
I am going to remove it, now.